android-app/app/build.gradle

```
@@ -46,6 +46,7 @@ android {
     }
     buildFeatures {
         viewBinding true
+        buildConfig = true
```

```suggestion
        buildConfig true
```

we don't use the "assignment" syntax anywhere in this file, could we use the "declarative" approach consistently?

---

yes it works. Good point, I didn't test it